Makefile

```
@@ -1066,20 +1066,15 @@ $(BUILD_PREFIX)$(TARGET_NAME).a: $(OBJS)
 	$(AR) rcs $(BUILD_PREFIX)$(TARGET_NAME).a $(filter-out $(ODIR)/main.o $(ODIR)/messages.o,$(OBJS))
 
 .PHONY: version prefix
+
+PYTHON != python3 -V
+ifeq ($(.SHELLSTATUS),0)
+  PYTHON := python3
+else
+  PYTHON := python
+endif
```

this prints
```
make: python3: No such file or directory
```
to stderr if python3 is not found.
I think it would be nicer to for the stderr, (and a bit less brittle wrt .SHELLSTATUS) if we can just use `which` to determine the "path to executable" a bit more directly. I.e.

```suggestion
PYTHON != which python3
ifeq ($(PYTHON),)
  PYTHON := python
endif
```